js/libraries/jbox/jBox.css

```diff
@@ -19,7 +19,6 @@
 .jBox-container {
 	background: #fff;
 	border:1px solid #ffbb00;
-	max-width:180px;
```

Comment:
if I extract the .jBox-container rules from  jBox.css to main.css completely it don't work. So lets keep it there for now, since I don't added this lines it in the jBox.css in this PR.